qutebrowser/browser/commands.py

```diff
@@ -1513,7 +1513,8 @@ def download(self, url=None, *, mhtml_=False, dest=None):
             )
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def view_source(self, edit=False):
+    @cmdutils.argument('pygment')
+    def view_source(self, edit=False, pygment=False):
         """Show the source of the current page in a new tab.
 
         Args:
```

Comment:
You should document `pygment`/`pygments` here.  Please also mention that Pygments is always used on QtWebKit.